website/src/content/docs/pt-br/linter/index.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Linter
+description: Como utilizar o Linter do Biome.
+---
+
+import NumberOfRules from "@src/components/generated/NumberOfRules.astro";
+import RecommendedRules from "@src/components/generated/RecommendedRules.astro";
+
+O Linter do Biome analisa de forma estática o seu código para encontrar erros comuns e ajudar você a escrever códigos idiomáticos.
+
+<NumberOfRules />
+
+## Usar o Linter via CLI
+
+Você pode começar executando a CLI para verificar possíveis erros usando o seguinte comando:
+
+```shell
+biome lint ./src
+```
+
+Para mais informações sobre todas as opções disponíveis, veja a página sobre [o CLI](/reference/cli#biome-lint)
+
+## Princípios das Regras
+
+Nós acreditamos que as regras devem ser informativas e que expliquem ao usuário porque uma regra é acionada, além de orientar o usuário sobre o que deve ser feito para corrigí-lo. Uma regra deve seguir esses **princípios**:
+
+1. Explicar o erro para o usuário. Geralmente, essa é a mensagem do diagnóstico.
+2. Explicar para o usuário porque o erro foi acionado. Geralmente, isso é implementado com um parâmetro adicional.
+3. Falar para o usuário o que ele precisa fazer. Geralmente, isso é implementado usando um "code action". Se o "code action" não for aplicável uma nota deve dizer o que o usuário precisa fazer para arrumar o erro.
```

Comment:
Any suggestions for translating "code action"?

Reply:
@CookieDasora "Ação no código" sounds good to you?